package.json

```diff
@@ -26,7 +26,8 @@
   },
   "scripts": {
     "postinstall": "node lib/install.js",
-    "test": "xo && ava"
+    "test": "xo && ava",
+    "build-linux": "docker build -t imagemin/mozjpeg docker; docker run --rm -v $(pwd)/vendor/linux:/src/out imagemin/mozjpeg cp cjpeg /src/out"
```

Comment:
Use full flags (not shortflags) wherever possible. That improves readability.

Reply:
`docker; docker` => `docker && docker`

It shouldn't try to run if building it fails.